discovery-deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: discovery-agent
+  labels:
+    name: discovery-agent
+    instance: discovery
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: discovery-agent
+      app.kubernetes.io/instance: discovery
+  template:
+    metadata:
+      annotations:
+      labels:
+        app.kubernetes.io/name: discovery-agent
+        app.kubernetes.io/instance: discovery
+    spec:
+#      imagePullSecrets:
+#        - name: <Secret_Name>
```

Comment:
Can this be deleted if its not used

Reply:
It is not used. However, if we are using test images on axway artifactory, we will need to populate this so I kept it for ease of testing.

Reply:
Once we move to helm, we can add an if conditional here